### DISCUSSION TITLE
wrong styling on iPhone

### DISCUSSION THREAD

**Discussion by sebsch1337** [neuefische/questions#300] [Web]
Dear people,

my app looks good on desktops with chrome (mac / pc) and android. But when i use iPhone (chrome / safari) my buttons are displayed differently. The boxes are smaller than the form. The drop shadow is cut at the boarders instead of fading out.
Below are some examples.

I'm using em units and set margin / padding to 0 for * in globalStyles.

How can i avoid this problem?

## How it should look like
![image](https://user-images.githubusercontent.com/75478893/199949699-b2851fe3-4c5e-4cf7-b8fc-40cec06d22a5.png)

## How it looks like on iPhone (box shadow cut // different box sizes)
![image](https://user-images.githubusercontent.com/75478893/199949573-a2e3693e-3f94-40c8-983c-6b3ea6b860fc.png)

**Comment by F-Kirchhoff** [comment]
Looks like the default values for rem/em on mobile safari are different. Please be aware that rem/em are not always the best option. Otherwise you can set the default value for rem in the ::root element in your globalstyles. 

Have a look into this wonderful article about the topic: https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

**Reply by sebsch1337** [reply]
Thanks for the info!

Do you also have an idea why the drop shadow is stuck in the box?

### Normal
![image](https://user-images.githubusercontent.com/75478893/199978318-25bbcb58-5dc7-481e-9132-96082c7acd02.png)

### iPhone / Safari
![image](https://user-images.githubusercontent.com/75478893/199978169-a9504d7c-5df7-4dc2-8cfb-3fa2a33f62ae.png)
